pkg/reconciler/taskrun/taskrun_test.go

```diff
@@ -4549,6 +4554,49 @@ func Test_validateTaskSpecRequestResources_InvalidResources(t *testing.T) {
 	}
 }
 
+func TestCreatePod_FeatureFlagVarsAreSet(t *testing.T) {
```

Comment:
It would be helpful to see a test with a TaskRun that has a mounted secret, and to check that the envvar set on the step containers has the correct value.

Reply:
I created a test for this: TestCreatePod_ExtractedSecretsAppliedToPodEnv